packages/android_alarm_manager/android/src/main/java/io/flutter/plugins/androidalarmmanager/AndroidAlarmManagerPlugin.java

```diff
@@ -147,21 +166,6 @@ public void onMethodCall(MethodCall call, Result result) {
     }
   }
 
-  /**
-   * Transitions the Flutter execution context that owns this plugin from foreground execution to
-   * background execution.
-   *
-   * <p>Invoked when the {@link FlutterView} connected to the given {@link FlutterNativeView} is
-   * destroyed.
-   *
-   * <p>Returns true if the given {@code nativeView} was successfully stored by this plugin, or
-   * false if a different {@link FlutterNativeView} was already registered with this plugin.
-   */
-  @Override
-  public boolean onViewDestroy(FlutterNativeView nativeView) {
```

Comment:
This still seems like it could be relevant. I think that view destruction is actually activity detachment. The question is, should the plugin hand over its FlutterEngine to the AlarmService when the Activity goes away? I'm not sure. But this plugin probably requires some careful consideration to its use-cases with the new embedding.

Reply:
It may have been relevant early on in this plugin's life, but I don't believe it is anymore. See my response to your other comment.